pyramid_mixpanel/track.py

```diff
@@ -16,8 +16,11 @@
 from pyramid_mixpanel.consumer import MockedConsumer
 from pyramid_mixpanel.consumer import PoliteBufferedConsumer
 
+import structlog
 import typing as t
 
+logger = structlog.get_logger(__name__)
```

Comment:
`structlog` shouldn't be a requirement of the package, it should be optional, like so: https://github.com/niteoweb/pyramid_heroku/blob/cdf99f5f3cba738872fd9b705b05bf0e700fa099/pyramid_heroku/herokuapp_access.py#L39